scss/civicrm/financial/_batchtransaction.scss

```diff
@@ -23,7 +23,7 @@
     }
 
     .form-layout-compressed {
-      input.crm-form-submit {
+      button.crm-form-submit {
         margin-left: 5px;
         padding: 4px 10px;
```

Comment:
In `_contributions.scss` we changed the padding values from `4px 10px` to `8px 11px`. Why wasn't the same change applied here?

Reply:
Good catch. the previous one also should be `4px 10px`. Not sure why I made it like that.